distribution/ddf-common/src/main/resources/etc/custom.properties

```diff
@@ -46,59 +46,218 @@ org.apache.karaf.security.providers=sun.security.provider.Sun,org.bouncycastle.j
 # Allow configuration admin to update config files
 felix.fileinstall.enableConfigSave = true
 
+#
```

Comment:
The extra packages were obtained from the karaf 4.0.4 distribution. I believe if we were to update to utilize the karaf-assembly maven target this would be generated for us (by specifying additional libraries).  This list basically states which packages should be available from the Framework (bundle 0) as provided by a library in the lib dir.
https://karaf.apache.org/manual/latest/developers-guide/custom-distribution.html (I'll write a ticket for this)

I would not expect an administrator to ever change this. An integrator may if they had a very specific need, but i would think that is unlikely as well.